common/ECSResourcesCheck/ReadMe.md

```diff
@@ -0,0 +1,76 @@
+#Automatic check and comparison of ECS cluster
```

Comment:
Please rename the file to "README.md" (all caps)

Reply:
done